### PR TITLE
Small cosmetic changes

### PR DESCRIPTION
**Pull request by gberaudo**
More or less depends on https://github.com/AnalyticalGraphicsInc/cesium/pull/5993.

### CODE REVIEW

File: src/olcs/contrib/manager.js [hunk 1]
```diff
@@ -335,4 +335,14 @@ olcs.contrib.Manager = class {
       });
     });
   }
+
+  /**
+   * @protected
+   * @return {Cesium.Rectangle|undefined}
+   */
+  getCameraExtentRectangle() {
```
Comment (fredj) [inline src/olcs/contrib/manager.js]: why not using a getter `get cameraExtentRectangle` ?

Comment (gberaudo) [reply src/olcs/contrib/manager.js]: I tried on another class and got an error from GCC. (something like export is only allowed for global methods and properties).

Comment (fredj) [reply src/olcs/contrib/manager.js]: ok

**Comment by gberaudo** [comment]
@fredj, I rebase and enabled the camera workaround by default. Does it still looks good to you?

**Comment by gberaudo** [comment]
Thanks for the reviews @fredj.

**Pull request merged**